beacon_node/http_api/src/lib.rs

```diff
@@ -2186,6 +2186,53 @@ pub fn serve<T: BeaconChainTypes>(
             },
         );
 
+    // POST validator/prepare_beacon_proposer
+    let post_validator_prepare_beacon_proposer = eth1_v1
+        .and(warp::path("validator"))
+        .and(warp::path("prepare_beacon_proposer"))
+        .and(warp::path::end())
+        .and(not_while_syncing_filter.clone())
+        .and(chain_filter.clone())
+        .and(warp::addr::remote())
+        .and(log_filter.clone())
+        .and(warp::body::json())
+        .and_then(
+            |chain: Arc<BeaconChain<T>>,
+             client_addr: Option<SocketAddr>,
+             log: Logger,
+             preparation_data: Vec<ProposerPreparationData>| {
+                blocking_json_task(move || {
+                    let execution_layer = chain
+                        .execution_layer
+                        .as_ref()
+                        .ok_or(BeaconChainError::ExecutionLayerMissing)
+                        .map_err(warp_utils::reject::beacon_chain_error)?;
+                    let current_epoch = chain
+                        .epoch()
+                        .map_err(warp_utils::reject::beacon_chain_error)?;
+
+                    info!(
```

Comment:
```suggestion
                    debug!(
```

This might get a little noisy when polling once per slot (per validator client).

Reply:
Agree 👍
Should we change the [log entry on VC side](https://github.com/pk910/lighthouse/blob/3072e5a25938a6f01003179928d9bbd72c27be04/validator_client/src/preparation_service.rs#L265-L269) to `debug`, too?

Reply:
Oh yep, good idea!